code/modules/modular_computers/file_system/programs/sm_monitor.dm

```diff
@@ -44,7 +44,7 @@
 	//var/valid_z_levels = (GetConnectedZlevels(T.z) & using_map.station_levels)
 	for(var/obj/machinery/power/supermatter_shard/S in GLOB.machines)
 		// Delaminating, not within coverage, not on a tile.
-		if(!(S.z in GLOB.station_z_levels) || S.z == ZLEVEL_MINING || S.z == T.z || !istype(S.loc, /turf/))
+		if(!((S.z in GLOB.station_z_levels) || S.z == ZLEVEL_MINING || S.z == T.z) || !istype(S.loc, /turf/))
```

Comment:
Why not throw the istype in and remove the inversion as well?

Reply:
make the istype an isturf instead too